scripts/restore/restore-harbor-job.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+ name: restore-harbor-job
+spec:
+ template:
+   metadata:
+     name: restore-harbor
+   spec:
+     volumes:
+      - name: scripts
+        configMap:
+          name: restore-harbor
+      - name: backup
+        emptyDir: {}
+     securityContext:
+      runAsUser: 1000
+      runAsGroup: 1000
+      fsGroup: 1000
+     restartPolicy: OnFailure
```

Comment:
Maybe?
```suggestion
     restartPolicy: Never
```

Reply:
Yeah maybe makes more sense when it is a single time job.